.github/workflows/dt.yaml

```diff
@@ -0,0 +1,52 @@
+name: DefinitelyTyped
+on: push
+jobs:
+  DefinitelyTyped:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Recurly-js
+        uses: actions/checkout@v2
+        with:
+          path: recurly-js
+      - name: Checkout DefinitelyTyped
+        uses: actions/checkout@v2
+        with:
+          repository: dbrudner/DefinitelyTyped
+          ref: master
+          path: DefinitelyTyped
+          fetch-depth: 1
+      # - name: Set remote
+      #   run: |
+      #     cd DefinitelyTyped
+      #     git config user.password ${{ secrets.DEFINITELY_TYPED_PR_GITHUB_ACCESS_TOKEN }}
+      #     git remote set-url origin https://github.com/dbrudner/DefinitelyTyped
+      #     git fetch --unshallow -p origin
+      - name: Move files
+        run: |
+          rm DefinitelyTyped/types/recurly__recurly-js/index.d.ts
+          rm -rf DefinitelyTyped/types/recurly__recurly-js/lib
+          rm -rf DefinitelyTyped/types/recurly__recurly-js/test
+          mkdir DefinitelyTyped/types/recurly__recurly-js/test
+          mv recurly-js/types/* DefinitelyTyped/types/recurly__recurly-js
+          mv recurly-js/test/types/* DefinitelyTyped/types/recurly__recurly-js/test
+          rm DefinitelyTyped/types/recurly__recurly-js/test/tsconfig.json
+          rm DefinitelyTyped/types/recurly__recurly-js/test/index.d.ts
```

Comment:
I think this can be simplified down to 

```bash
cp -R recurly-js/types/ DefinitelyTyped/types/recurly__recurly-js
cp -R recurly-js/test/types/ DefinitelyTyped/types/recurly__recurly-js/test
```

Those `cp` commands may be slightly off